src/components/pages/Navbar/NavbarFeatures/LogoutButton.js

```diff
@@ -1,24 +1,20 @@
 import React from 'react';
 import { useAuth0 } from '@auth0/auth0-react';
-
 import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
 
 function LogoutButton() {
   const { logout, isAuthenticated } = useAuth0();
 
   const logoutAuth = () => {
     localStorage.removeItem('AuthToken');
-    logout();
+    logout({ returnTo: window.location.origin });
   };
 
   return (
     isAuthenticated && (
-      <button
-        className="auth logout"
-        onClick={() => logout({ returnTo: window.location.origin })}
-      >
+      <NavBtnLink className="auth logout" onClick={logoutAuth}>
```

Comment:
This is another good change to make the code for the logout button more concise, while incorporating the styling that has been created for the buttons.